score.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SCORE_H
+#define SCORE_H
+
+#include <string>
+#include <fstream>
+#include <filesystem>
+#include <vector>
+
+class Score{
+public:
+    Score();
+    Score(std::string specialPath);
+    int getBestScore(){return bestScore;}
+    //void setScoreFile(std::string specialPath){specialPath = way;}
+    void saveCurrentScoreToFile();
```

Comment:
ToFile is redundant information here. One day I might ask you to save it to database and would you then rename it with ToDatabase everywhere you call this function